### PR TITLE
fix: flyway拼写错误格式化问题修改

### PR DESCRIPTION
**Pull request by ziyujiahao**
#### Please indicate you've done the following:

- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Made sure commit message follow the rule of [Conventional Commits specification](https://www.conventionalcommits.org/).
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed.

### CODE REVIEW

File: backend/src/main/resources/db/migration/V40__1.15.sql [hunk 1]
```diff
@@ -213,7 +213,7 @@ delimiter ;
 DROP FUNCTION IF EXISTS `GET_CHART_VIEW_COPY_NAME`;
 delimiter ;;
 CREATE FUNCTION `GET_CHART_VIEW_COPY_NAME`(chartId varchar(255),pid varchar(255))
-    RETURNS varchar(255) CHARSET utf8mb4
+    RETURNS varchar(255) CHARSET utf8mb4 COLLATE utf8mb4_general_ci
```
Comment (xuwei-fit2cloud) [inline backend/src/main/resources/db/migration/V40__1.15.sql:216]: 还有几处 CHARSET utf8mb4 没有加上 COLLATE utf8mb4_general_ci

**Review by xuwei-fit2cloud — COMMENTED** [review]
还有几处 CHARSET utf8mb4 没有加上 COLLATE utf8mb4_general_ci

File: backend/src/main/resources/db/migration/V40__1.15.sql (hunk 1 above)
Comment (ziyujiahao) [reply backend/src/main/resources/db/migration/V40__1.15.sql:216]: 已修改

**Review by xuwei-fit2cloud — APPROVED** [review]
LGTM

**Pull request merged**